content/languages/ruby/index.md

```diff
@@ -18,9 +18,9 @@ From Ruby 3.0, [RSpec](https://rspec.info/) is used under the hood:
 # Defaults to the global `describe` for backwards compatibility, but `RSpec.desribe` works as well.
 describe "Example" do
   it "should return the sum" do
-    expect(add(1, 1)).to eq(2)
+    expect(add(1, 1)).to eq(2), "failure message"
     # The following is still supported, but new tests should not use them.
-    # Test.assert_equals(add(1, 1), 2)
+    # Test.assert_equals(add(1, 1), 2, "failure message")
   end
 end
 ```
```

Comment:
````suggestion
```
Note that if an assertion message is provided while using RSpec, actual and expected results have to be in that message otherwise the information will not be shown to the user.
````